script.js

```diff
@@ -7,32 +7,38 @@ let books = [];
 function displayData() {
   let bookList = '';
   for (let i = 0; i < books.length; i += 1) {
-    bookList += `<div class=book>
-    <h3 class="title">${books[i].title}     
-      </h3>
-      <p class="author">
-      ${books[i].author}</p>
-      <div class="remove"><button onClick="deleteBook(${i})">Remove</button>
-      </div>
-      <hr>
-      <br>
+    bookList += `<div class="book">
+   <div>
+    <td>${books[i].title}     
+      </td>
+      <td> by </td>
+      <td>
+      ${books[i].author}</td></div>
+      <div>
+      <td><button onClick="deleteBook(${i})" class="remove">Remove</button>
+      </td></div>
      </div>
      `;
   }
   document.getElementById('tdBody').innerHTML = bookList;
 }
 
-function add() {
+class book {
+  constructor(title, author) {
```

Comment:
- [ ] I think it would be better if you capitalize the class name (`class book { ... }`) to follow a good [JavaScript name convention](https://www.robinwieruch.de/javascript-naming-conventions/).